### PR TITLE
[DOCS-4776]: AWS org metrics

### PR DESCRIPTION
**Pull request by aurbiztondo-splunk**
New org metrics for AWS, see jira.

### CODE REVIEW

File: signalfx-org-metrics/metrics.yaml [hunk 1]
```diff
@@ -378,6 +378,46 @@ sf.org.datapointsTotalCountByToken:
         * Data resolution: 10 seconds
   metric_type: counter
   title: sf.org.datapointsTotalCountByToken  
+ 
+sf.org.datainventoryAdded:
```
Comment (penglerts) [inline signalfx-org-metrics/metrics.yaml:382]: `sf.org.datainventory.datapointsAdded`

File: signalfx-org-metrics/metrics.yaml [hunk 2]
```diff
@@ -378,6 +378,46 @@ sf.org.datapointsTotalCountByToken:
...
+  brief: DEF
+  description: |
+    DEF
+    
+        * Dimension(s): `orgId`, ?
```
Comment (penglerts) [inline signalfx-org-metrics/metrics.yaml:387]: Same dimensions for all of them
- type: eq. `AWS`
- service
- region
- integrationId

File: signalfx-org-metrics/metrics.yaml [hunk 3]
```diff
@@ -378,6 +378,46 @@ sf.org.datapointsTotalCountByToken:
...
+        * Data resolution: ?
+  metric_type: ?
```
Comment (penglerts) [inline signalfx-org-metrics/metrics.yaml:389]: count

File: signalfx-org-metrics/metrics.yaml [hunk 4]
```diff
@@ -378,6 +378,46 @@ sf.org.datapointsTotalCountByToken:
...
+  title: sf.org.datainventoryAdded
+  
+sf.org.datainventory.latestTimestamp:
+  brief: DEF
+  description: |
+    DEF
+    
+        * Dimension(s): `orgId`, ?
+        * Data resolution: ?
+  metric_type: ?
```
Comment (penglerts) [inline signalfx-org-metrics/metrics.yaml:399]: gauge

File: signalfx-org-metrics/metrics.yaml [hunk 5]
```diff
@@ -378,6 +378,46 @@ sf.org.datapointsTotalCountByToken:
...
+  title: sf.org.datainventory.latestTimestamp 
+  
+sf.org.datainventory.mtses:
+  brief: DEF
+  description: |
+    DEF
+    
+        * Dimension(s): `orgId`, ?
+        * Data resolution: ?
+  metric_type: ?
```
Comment (penglerts) [inline signalfx-org-metrics/metrics.yaml:409]: gauge

File: signalfx-org-metrics/metrics.yaml [hunk 6]
```diff
@@ -378,6 +378,46 @@ sf.org.datapointsTotalCountByToken:
...
+  title: sf.org.datainventory.mtses 
+  
+sf.org.datainventory.resources:
+  brief: DEF
+  description: |
+    DEF
+    
+        * Dimension(s): `orgId`, ?
+        * Data resolution: ?
+  metric_type: ?
```
Comment (penglerts) [inline signalfx-org-metrics/metrics.yaml:419]: gauge

File: signalfx-org-metrics/metrics.yaml (hunk 1 above)
Comment (aurbiztondo-splunk) [reply signalfx-org-metrics/metrics.yaml]: Thanks! Fixed!

File: signalfx-org-metrics/metrics.yaml (hunk 2 above)
Comment (aurbiztondo-splunk) [reply signalfx-org-metrics/metrics.yaml]: Added!

File: signalfx-org-metrics/metrics.yaml [hunk 7]
```diff
@@ -378,6 +378,45 @@ sf.org.datapointsTotalCountByToken:
         * Data resolution: 10 seconds
   metric_type: counter
   title: sf.org.datapointsTotalCountByToken  
+ 
+sf.org.datainventory.datapointsAdded:
+  brief: DEF
+  description: |
+    It's **only** generated when metrics are pushed with Metric Streams.
```
Comment (penglerts) [inline signalfx-org-metrics/metrics.yaml:385]: not only for metric streams

**Review by theletterf — APPROVED** [review]

**Pull request merged**